vendor/github.com/MustWin/baremetal-sdk-go/url_builder.go

```diff
@@ -42,6 +42,9 @@ func buildDatabaseURL(urlTemplate string, region string, resource resourceName,
 func buildObjectStorageURL(urlTemplate string, region string, resource resourceName, query url.Values, ids ...interface{}) string {
 	baseUrl := baseUrlHelper(urlTemplate, objectStorageServiceAPI, region)
 	urlStr := fmt.Sprintf("%s/%s", baseUrl, resourceNamespaces)
+	if resource == resourcePAR {
+		return buildURL(urlStr, make(map[string][]string), ids...)
+	}
```

Comment:
I think this got dropped from the sdk code review, an inline comment on this would be useful, also since the original query value is getting overwritten, perhaps this should be documented at the sdk function as well.